pyprof/nvtx/nvmarker.py

```diff
@@ -159,10 +165,10 @@ def get_trace_info(op_name):
         stack = traceback.extract_stack()
         func_stack = ""
 
-        # Previous frame name and line. This is the file and line 
+        # Previous frame name and line. This is the file and line
         # that CALLED the frame we are in
         #
-        prev_fnl = "" 
```

Comment:
why are there so many whitespace changes? Almost all of the changes up to this point are whitespace